### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by nanego**

### CODE REVIEW

File: spec/exp_descriptor_fr_spec.rb [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    it 'time of day with seconds' do
+      expect(desc_fr('30 02 14 * * *')).to eq('À 2:02:30 PM')
+    end
+
+    it 'second intervals' do
+      expect(desc_fr('5-10 * * * * *')).to eq('Secondes 5 à 10 après la minute')
+    end
+
+    it 'second minutes hours intervals' do
+      expect(desc_fr('5-10 30-35 10-12 * * *')).to eq("Secondes 5 à 10 après la minute, minutes 30 à 35 après l'heure, entre 10:00 AM et 12:59 PM")
```
Comment (alpinweis) [inline spec/exp_descriptor_fr_spec.rb]: nit: can you move the string to the next line

File: resources/resources_fr.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# resources_fr_FR.yml
```
Comment (alpinweis) [inline resources/resources_fr.yml]: nit: ` resources_fr.yml`

File: resources/resources_fr.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+march:     'mars'
+april:     'avril'
+may:       'mai'
+june:      'juin'
+july:      'juillet'
+august:    'août'
+september: 'septembre'
+october:   'octobre'
+november:  'novembre'
+december:  'décembre'
```
Comment (alpinweis) [inline resources/resources_fr.yml]: quotes are not really necessary for these guys

**Comment by alpinweis** [comment]
please also bump up the version and update the changelog

**Comment by nanego** [comment]
Hello.
Here are some improvements.
Thank you
Vincent

**Pull request merged**